### PR TITLE
FBX-487: Bump package version to 5.1.0

### PR DESCRIPTION
**Pull request by windxu88**
## Purpose of this PR:
Bump package version to 5.1.0

JIRA ticket:
[FBX-487](https://jira.unity3d.com/browse/FBX-487) 5.1.0 - Open package release PR(s)

### CODE REVIEW

**Review by lucillecaillaud — APPROVED** [review]
It's annoying to have an empty CHANGELOG, but I don't know what to do about that.
Maybe Seb will have an idea.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # Changes in FBX SDK C# Bindings
 
+## [5.1.0] - 2023-09-08
+
```
Comment (sebastienduverne) [inline CHANGELOG.md:4]:
```suggestion

* Updated package version number for release.
```

**Review by sebastienduverne — COMMENTED** [review]
Changelog suggestion

**Comment by windxu88** [comment]
@sebastienduverne I have committed the change you suggested. If there is nothing else, I will merge this PR. Thanks

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,8 @@
 # Changes in FBX SDK C# Bindings
 
+## [5.1.0] - 2023-09-08
+
+* Updated package version number for release.
```
Comment (sebastienduverne) [inline CHANGELOG.md:5]:
```suggestion
* Updated package version number for release.

```

**Review by sebastienduverne — APPROVED** [review]
Minor thing (vertical space) I didn't see in the first pass – sorry –, I think it's working without it but the source will be cleaner.
And yes, that's the only update we can do for this PR so I'm approving, you can merge after this last tiny fix.

**Pull request merged**